session_rpcserver.go

```diff
@@ -984,6 +984,19 @@ func (s *sessionRpcServer) AddAutopilotSession(ctx context.Context,
 					name)
 			}
 			configB = af.Configuration
+		} else {
+			configB = f.Config
+		}
+
+		// The configuration may contain sensitive data like pubkeys,
+		// which we replace here and add to to the privacy map.
+		if privacy {
+			configB, err = firewall.ObfuscateConfig(
+				privacyMapPairs, configB,
+			)
+			if err != nil {
+				return nil, err
+			}
```

Comment:
this means that `featureConfig` given to `NewSession` and thus persisted, will be the obfuscated one right? Dont think we want to do that.

Reply:
very good point :+1: ! the new version persists the unobfuscated data